src/main/scala/com/timushev/sbt/updates/MetadataLoader.scala

```diff
@@ -11,16 +12,27 @@ import scala.concurrent.Future
 import scala.xml.{Elem, XML}
 
 object MetadataLoaderFactory {
-  def loader(logger: Logger): PartialFunction[Resolver, MetadataLoader] = {
-    case repo: MavenRepository => new MavenMetadataLoader(repo, download(logger))
+  def loader(logger: Logger, credentials: Seq[Credentials]): PartialFunction[Resolver, MetadataLoader] = {
+    case repo: MavenRepository =>
+      val repoCredentials = Credentials.forHost(credentials, new URL(repo.root).getHost)
+      new MavenMetadataLoader(repo, download(logger, repoCredentials))
   }
 
   private val cache = mutable.Map[String, Future[Elem]]()
 
-  def download(logger: Logger)(url: String) = synchronized {
+  def download(logger: Logger, credentials: Option[DirectCredentials])(url: String) = synchronized {
     cache.getOrElseUpdate(url, Future {
-      logger.debug(s"Downloading $url")
-      XML.load(new URL(url))
+      credentials match {
+        case Some(c) =>
+          val auth = Base64.Encoder(s"${c.userName}:${c.passwd}".getBytes).toBase64
```

Comment:
Can you please use `javax.xml.bind.DatatypeConverter#printBase64Binary(byte[])` instead? I'd prefer to avoid using external libraries to avoid classpath conflicts with other plugins.